velox/vector/arrow/Bridge.cpp

```diff
@@ -582,13 +583,15 @@ VectorPtr createStringFlatVectorFromUtf8View(
   // buffer-index, 4-byte buffer-offset] to 16-byte Velox StringView [4-byte
   // length, 4-byte prefix, 8-byte buffer-ptr]
   for (int32_t idx_64 = 0; idx_64 < arrowArray.length; ++idx_64) {
-    auto* view = (uint32_t*)(&((uint64_t*)arrowArray.buffers[1])[2 * idx_64]);
-    rawStringViews[2 * idx_64] = *(uint64_t*)view;
+    auto* view = reinterpret_cast<const uint32_t*>(&(
```

Comment:
Why add const here and elsewhere in the block instead of keeping it non-const?
Compared to line 783 change as well for example.

Reply:
arrwoArray.buffers is a const void** so the reinterpret_cast has error if we lose the constness. Same in other places where const is added.